packages/launchpad-ledger/src/launchpadledger.ts

```diff
@@ -2,7 +2,7 @@ import { HdPath, Secp256k1Signature } from "@cosmjs/crypto";
 import { fromUtf8 } from "@cosmjs/encoding";
 import { makeCosmoshubPath } from "@cosmjs/launchpad";
 import { assert } from "@cosmjs/utils";
-import Transport from "@ledgerhq/hw-transport";
+import LedgerTransport from "@ledgerhq/hw-transport";
```

Comment:
Why rename this interface?

Reply:
`Transport` is already available via the DOM lib: https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L20083

Reply:
And we cannot override it? If I recall correctly, importing types overrides globals

Reply:
We can. Do we... want to?

Reply:
I think we should. This is only a type and not a global object. `Transport` is undefined in JavaScript (node.js, Firefox and Chrome)